packages/elements/src/icon/__test__/helpers/helpers.js

```diff
@@ -1,24 +1,61 @@
-import { fixture, nextFrame } from "@refinitiv-ui/test-helpers";
+import { fixture, nextFrame, aTimeout } from "@refinitiv-ui/test-helpers";
 
 export const tickSvg = '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path stroke="#000" d="M14 4l-8.25 8.25L2 8.5" fill="none" fill-rule="evenodd"></path></svg>';
 export const iconName = 'tick';
 let iconId = 0;
 
 export const createAndWaitForLoad = async template => {
-    const el = await fixture(template);
-    await nextFrame();
-    return el;
+  const el = await fixture(template);
+  await aTimeout(50);
+  await nextFrame();
+  return el;
 };
 
 export const checkRequestedUrl = (requests, url) =>{
-    for (let i = 0; i < requests.length; i++) {
-      if(requests[i].url === url){
-        return true;
-      }
+  for (let i = 0; i < requests.length; i++) {
+    if(requests[i][0] === url){
+      return true;
     }
-    return false;
+  }
+  return false;
 }
 
 export const generateUniqueName = name => `${name}_${iconId+=1}`;
 
 export const createMockSrc = icon => `https://mock.cdn.com/icons/${icon}.svg`;
+
+export const createFakeResponse = (body, config = responseConfigSuccess) => {
```

Comment:
I have no idea what this function is used for. How it related to test case?

Reply:
In test, we don't make real request. The test must work in offline. That why I create this function to mock response to make it as fetch response.